tests/test_namespace.py

```diff
@@ -0,0 +1,58 @@
+from contextlib import contextmanager
+
+from cms.api import create_page
+from django.test import Client
+from django.utils.encoding import force_str
+
+from .base import BaseTest
+from .test_utils.models import CustomUser as User
+
+try:
```

Comment:
this is not needed

Reply:
Agreed

Reply:
apparently the push with this change did not make it (I had myself issues with github in the past hour or so)

Reply:
Commandline says it is up to date.

I notice that github is changing its ip every few minutes...

Reply:
github cowardly refusing to get your commit, apparently :roll_eyes: 

nevermind, I will merge as is and we will cleanup later